src/screens/Main.tsx

```diff
@@ -34,14 +34,67 @@ interface Props {
   className?: string;
 }
 
+interface MenuActionItemsProps {
+  idx: number;
+  title: string;
+  isEnabled: boolean;
+  component: React.ReactElement;
+}
+
+const MenuContents = [
+  {
+    idx: 0,
+    title: 'Transfer',
+    isEnabled: true,
+    component: (
+      <>
+        <Transfer />
+        <ExtensionAccountCheck component={<Accounts />} />
```

Comment:
Why do we check extension/account only on tranfser page? It is required for remark or custom call as well.